### PR TITLE
Home and About page

### PR DESCRIPTION
**Pull request by Bettenoch**
**In this branch I made the following changes:**

- [x] ♻Add home and about pages
- [x] ♻Created speakers section in home page dynamically using Javascript  file.
- [x] ♻Style home page and about page.
- [x] ♻Add harmburger and close menu for mobile version
- [x] ♻Personalized content to suite a web development course.
- [x] ♻Add linters file
- [x] ♻Add and updated readme file.

For a live demo of the project please visit [here](https://www.loom.com/share/44c9101ccb47414b8a43f64923d7f558)

Thank you for reviewing my repository!!!🙂

### CODE REVIEW

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @Bettenoch, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have all linters check passed. please make sure you fix the issues before asking for another request.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Gambit142 — COMMENTED** [review]
Hi @Bettenoch 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- The linters setup that you used was for `Javascript` projects. Kindly use the one for `HTML/SS + JavaScript` projects: https://github.com/microverseinc/linters-config/tree/master/html-css-js

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Bettenoch** [comment]
Hello @Gambit142 thank you for assistance. Concerning the change shoudl I reinstall the Linters files?

**Comment by Gambit142** [comment]
Hi @Bettenoch 👋,

I think that will be the best thing to do. 👍

**Comment by Bettenoch** [comment]
I changed the linters file I really hope it is okay now

File: README.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+- **[Cross-plartfom responsiveness]**
+- **[User Friendliness]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://bettenoch.github.io/Portfolio-Setup-Mobile-First)
```
Comment (mckent05) [inline README.md]: - You have added the wrong link for your capstone project, I suggest you deploy your project live and add the correct link to your Readme in order to make it more professional 👍 👍

File: README.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+- Thanks to the Microverse team for the great curriculum.
+- Thanks to the Code Reviewer(s) for the insightful feedbacks.
+- A great thanks to My coding partner(s), morning session team, and standup team for their contributions.
+
```
Comment (mckent05) [inline README.md]: - According to the project requirements, you are required to give credit to [Cindy Shin](https://www.behance.net/adagio07) the author of the design you have used, please fix this 🙏

File: index.html [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+                </article>
+                <article class="header-element third">
+                    <h3>2023.10.15(THU) ~ 16(FRI)</h3>
+                    <h4>
+                        @ Kenya National Institute of Conference (KICC), at Nairobi.
+                    </h4>
+                </article>
+            </article>
+            
+        </article>
```
Comment (mckent05) [inline index.html:91]: - You have done well so far, but you have not styled this page according to the design template, I suggest that the background blend for this section should cover the entire section and not just a part of it. Please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

- Also the font-color for the text I highlighted in the image below is not the same as seen in the design template. Please note that, though you are required to personalize the content of the page, you are required to stick to the design layout, background colors, font-style etc. Please fix this 🙏 
Your project:
- Please try to replicate the design template as closely as possible 😄 
yw4w

Design template:
rwegwrg

File: index.html [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+                    various countries about the
+                     messages of sharing and opening.
+                </p>
+            </div>
+        </article>
+        <article class="program_footer">
+            <h5>SEE THE WHOLE PROGRAM</h5>
+        </article>
+      </article>  
+    </section>
```
Comment (mckent05) [inline index.html]: - You have done well so far, but please remember to stick to the design template in terms of layout, spacing, borders, font color, background color, etc. The background color for this section is not the same as the given design template. Please fix this for both mobile and desktop versions🙏 

- In the mobile version, the section marked in the image below is not styled as it is in the given design template, please fix this 🙏. It is important to replicate the design template to give confidence to recruiters in our ability 👍 
Your project:
sfjwrsgwr

Design template:
rergerges

File: index.html [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+                        </p>
+                        <p class="speaker-description">
+                            As the main venue for new media art production in Korea,
+                             Nabi promotes cross-disciplinary collaboration 
+                             and understanding among science 
+                             technology, humanities, and the arts.
+                        </p>
+                    </div>
+                </li>
+            </ul>
```
Comment (mckent05) [inline index.html]: - According to the project requirements, you are to ensure that the entire speaker section is rendered dynamically using JavaScript, please remove all HTML templates for this section and render using only Javascript 👍 👍. Please fix this 😄  
- Also, please ensure that the images for the individual speakers are styled according to the design template. I suggest making them a little bit bigger. Please fix this for both mobile and desktop version 👍 👍 
Your project:
trergrk

Design template:
wfefwae

File: index.html [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+                        <a href="#" class="text-border"></a>
+
+                    </h2>
+                    <ul class="partnerr-logo">
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/googlekorea 1.svg" alt="google"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/daumkakaologo 1.svg" alt="microsoft"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/airbnb 1.svg" alt="Bolt"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/mozilla 1.svg" alt="Mozilla"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/airbnb 1.svg" alt="google"></span></li>
+                        <li class="company1-logo"><span> <img class="img2-logo" src="./assets/images/googlekorea 1.svg" alt="microsoft"></span></li>
```
Comment (mckent05) [inline index.html]: - Good job done so far, but please it is important to stick to the layout in the original design, you have added too many sponsors in your sponsor section. I suggest you style and design this section accordingly. Please fix this 😄 
Your project:
erreev

Design template:
wefefwf

File: about.html [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+                                 <p class="description">An owesome content creator, web developer and alumni at BKE COURSE</p>
+                                 <button class="bdn">view more</button>
+                             </div>
+                         </div>
+                    </div> 
+                 </div>
+            </article>
+            
+        </article>
+    </section>
```
Comment (mckent05) [inline about.html]: - Good job done so far, but please note that as much as you are required to personalize the content of your project, it is still important to stick to the design layout, spacing, colors etc. There are two missing sections in your project, the Logo section and the `previous summit section`. Please, follow strictly the layout of the design template to give confidence to recruiters in your ability to replicate exact business requirements. Please fix this 🙏 
Your project:
-Please design for both desktop and mobile versions accordingly 👍 👍 
erergergke

Design template:
wegqeg

- Also, on the mobile version of the About page, the `partner` section is missing, please include this section in your design 🙏 
Design template:
rgqwg3

File: about.html [hunk 8]
```diff
@@ -0,0 +1,281 @@
...
+                    <ul class="partner-logo">
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/google.svg" alt="google"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Microsoft.svg" alt="microsoft"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Bolt_logo.svg.png" alt="Bolt"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/mozilla.svg" alt="Mozilla"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/google.svg" alt="google"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Microsoft.svg" alt="microsoft"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/Bolt_logo.svg.png" alt="Bolt"></span></li>
+                        <li class="company-logo"><span> <img class="img-logo" src="./assets/images/mozilla.svg" alt="Mozilla"></span></li>
```
Comment (mckent05) [inline about.html]: - Please note that to ensure your HTML is readable and easy to maintain, it is not advisable to leave commented-out code on your page. Please ensure that only the needed code or HTML template is present on your page 👍

File: about.html [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+                        the ability to learn the skills they’d need to succeed in the 21st century. 
+                        We set out to create a new, interactive way of learning — making it engaging, 
+                        flexible, and accessible for as many people as possible. 
+                        Since then, we have helped millions of people worldwide unlock 
+                        modern technical skills and reach their full potential through code. ✨
+                    </p>
+                    <small>Made with ❤️in NYC © 2023 BWD</small>
+                   
+                </div>
+            </article>
```
Comment (mckent05) [inline about.html]: - You have not styled this section of your project correctly, please note that in the mobile version, the footer has a dark background color, while in the desktop version, the footer has a white background color. Please fix this 🙏
Your project:
Mobile version:
ewefwefwef
Desktop version:
eruerer

Design template:
Mobile version:
gergerv
Desktop version:
gwfwg

File: about.html [hunk 10]
```diff
@@ -0,0 +1,281 @@
...
+                    Transform you skils and build stunning, user-friednly Studies
+                    that are compatible with desktop computers
+                    , tablets, and mobile phones.
+                    Learn the fundamentals of web developmemt with HTML, CSS, and Javascript,
+                    as you create responsive web pages and robus web applications.
+                    Transform you skils and build stunning, user-friednly Studies
+                    that are compatible with desktop computers
+                    , tablets, and mobile phones.
+                </p>
+            </article>
```
Comment (mckent05) [inline about.html:83]: - Good job done so far, but please style this section of the page according to the design template. Please stick to the layout, font color, spacing, background color etc. please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Bettenoch, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍
- Good job working on your capstone project, keep going 🚀  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
**OPTIONAL** - To help improve the UI and look of your site, I suggest you do not display the logo on the mobile menu. Please consider this 🙏 
Your project:
weawewe

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bettenoch** [comment]
Hello @mckent05 thank you for you review comments. On the Logo section and the previous summit sections I changed the text to suite my content for a web development course. I was wondering if it is a must I use a logo instead of another image.

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi  @Bettenoch    👋

Your project is complete! ✔️✔ , I loved your work, it looks great 🔥🔥🔥🔥 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors

- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**